ipatests/test_integration/test_sssd.py

```diff
@@ -432,15 +432,47 @@ def test_subdomain_lookup_with_certmaprule_containing_dn(self):
             "--matchrule='<ISSUER>{}'".format(cert_subject),
             "--domain={}".format(self.master.domain.name)
         ])
-        tasks.clear_sssd_cache(self.master)
+        try:
+            tasks.clear_sssd_cache(self.master)
+            # verify the user can be retrieved after the certmaprule is added
+            second_res = self.master.run_command(
+                ['id', self.users['ad']['name']])
+            assert first_res.stdout_text == second_res.stdout_text
+            verify_in_stdout = ['gid', 'uid', 'groups',
+                                self.users['ad']['name']]
+            for text in verify_in_stdout:
+                assert text in second_res.stdout_text
+        finally:
+            self.master.run_command(
+                ['ipa', 'certmaprule-del', "'{}'".format(cert_subject)])
```

Comment:
Can you extract the fix for this test `test_subdomain_lookup_with_certmaprule_containing_dn` in a separate commit and refer to https://pagure.io/SSSD/sssd/issue/3721 ?